continuous_integration/environment.yaml

```diff
@@ -0,0 +1,30 @@
+name: test-environment
+channels:
+  - conda-forge
+dependencies:
+  - xarray
+  - dask
+  - distributed
+  - toolz
+  - Cython
+  - sphinx
+  - cartopy
+  - pillow
+  - matplotlib
+  - scipy
+  - pyyaml
+  - pyproj
+  - pyresample
+  - coveralls
+  - coverage
+  - codecov
+  - behave
+  - mock
+  - zarr
+  - geoviews
+  - pytest
+  - pytest-cov
+  - fsspec
+  - pip
+  - pip:
+    - trollsift
```

Comment:
If I remember right the conda-incubator used in the actions for making the environment doesn't support pip installations right now. If these are passing in your fork, then nevermind.

Reply:
Oh maybe it was mamba that didn't support it.

Reply:
I think I have done this before in pyspectral and it seemed to work...